yarn.lock

```diff
@@ -10339,10 +10339,10 @@ eth-block-tracker@^4.4.2:
     pify "^3.0.0"
     safe-event-emitter "^1.0.1"
 
-eth-contract-metadata@^1.11.0:
-  version "1.11.0"
-  resolved "https://registry.yarnpkg.com/eth-contract-metadata/-/eth-contract-metadata-1.11.0.tgz#4d23a8208d5d53be9d4c0696ed8492b505c6bca1"
-  integrity sha512-Bbvio71M+lH+qXd8XXddpTc8hhjL9m4fNPOxmZFIX8z0/VooUdwV8YmmDAbkU5WVioZi+Jp1XaoO7VwzXnDboA==
+eth-contract-metadata@^1.11.0, eth-contract-metadata@^1.12.0:
```

Comment:
Whoops! We should have probably updated this in GABA too.

Oh well - `eth-contract-metadata` is not used any of the GABA controllers the extension uses, so it's fine.

Reply:
Yeah this is fine